p2p/backend-test-suite/src/connect.rs

```diff
@@ -48,9 +43,6 @@ where
     .unwrap();
 }
 
-// Check that connecting twice to the same address isn't possible.
```

Comment:
Am I missing something or the comment is still relevant?

Reply:
Sorry, my bad, reverted.